### ISSUE TITLE
Restore python 2.7 support

### ISSUE THREAD

**Issue by rsheeter** [google/fonts#964]
04246627 broke our ability to run tools with python 2.7.

**Comment by graphicore** [comment]
What is the error message that you are getting? 

This is supposed to be compatible with python 2.7. 
I used `libprotoc 2.6.1` to generate the file, which is the default on my system

```
$ protoc --version
libprotoc 2.6.1
```

Maybe the `protoc` version I used has a bug? Though, I can at least import the `fonts_public_pb2` module without any errors with python 2.7

**Comment by graphicore** [comment]
The exception, reported to me on another channel, relevant part:

```
 from google.protobuf import symbol_database as _symbol_database
ImportError: cannot import name symbol_database
```

I made a gist for this: https://gist.github.com/graphicore/d80635b7a9164271720b5108474706c3

The import error goes away from version 2.6.0 onwards of the protobuf package (uploaded to PyPi on 2014-08-25).

**Comment by davelab6** [comment]
Upgrading libprotoc fixed our ability to run tools with python 2.7.
